library/.eslintignore

```diff
@@ -0,0 +1 @@
+template/
```

Comment:
Just curious? Why do we need this in `.eslintignore`? My guess would be that the template should validate as well.

(add a newline to the end of the file if we do need it)

Reply:
template has a .eslint file, referencing the .eslint file in the root